src/mongocxx/test/spec/util.cpp

```diff
@@ -56,7 +57,21 @@ uint32_t error_code_from_name(string_view name) {
     return 0;
 }
 
+/* Called with the entire test file and individual tests. */
 bool should_skip_spec_test(const client& client, document::view test) {
+    std::set<std::string> unsupported_tests = {
+        "CreateIndex and dropIndex omits default write concern",
```

Comment:
As described in https://jira.mongodb.org/browse/CXX-1351, there is a `create_index()` method, but not a  `drop_index()` on the collection class.

Reply:
Agreed, it makes syncing tests a lot simpler. The C driver has a similar list for tests of unsupported operations.